acceptance/tests/validate_vendored_ruby.rb

```diff
@@ -144,7 +145,7 @@ def install_dependencies(agent)
     agents.each do |agent|
       irb = irb_command(agent)
       ['hocon', 'deep_merge'].each do |gem|
-        stdout = on(agent, "echo \"require '#{gem}'\" | #{irb}").stdout.chomp
+        stdout = on(agent, "echo \"require '#{gem}'\" | #{irb}").stdout.chomp)
```

Comment:
I think this trailing paren is the source of the syntax error in the PR tests -- @melissa would you mind making an update? Other than that I'm excited to merge this

Reply:
I think there was a different reason we were dragging our feet on this, I can't quite remember it since it's been a while....

Reply:
Unless there's something I don't know about, I believe [this](https://github.com/puppetlabs/ci-job-configs/blob/master/resources/scripts/vanagon-integration.sh#L78) is the only place we'll need to update CI configuration to make this work -- @branan do you remember anything else?

Reply:
Yeah, the CI configuration is just that. And we really need to standardize this

Reply:
Oh right! Yeah, the conversation about needing to standardize the rake tasks. I updated this line, so if that's the only thing blocking this.... then we can standardize rake tasks at a later date